RecoMuon/Configuration/python/RecoMuonCosmics_cff.py

```diff
@@ -102,19 +105,16 @@
 # Sequences
 
 # Stand Alone Tracking
-STAmuontrackingforcosmics1Leg = cms.Sequence(CosmicMuonSeed*cosmicMuons1Leg)
+STAmuontrackingforcosmics1LegTask = cms.Task(CosmicMuonSeed,cosmicMuons1Leg)
 
 # Stand Alone Tracking plus global tracking
-muontrackingforcosmics1Leg = cms.Sequence(STAmuontrackingforcosmics1Leg*globalCosmicMuons1Leg)
-
-# all muons id
-allmuons1Leg = cms.Sequence(muons1Leg)
+muontrackingforcosmics1LegTask = cms.Task(STAmuontrackingforcosmics1LegTask, globalCosmicMuons1Leg)
 
 # Stand Alone Tracking plus muon ID
-STAmuonrecoforcosmics1Leg = cms.Sequence(STAmuontrackingforcosmics1Leg)
+STAmuonrecoforcosmics1Leg = cms.Sequence(STAmuontrackingforcosmics1LegTask)
```

Comment:
This (and the comment above) can probably be also cleaned out: it is identical to STAmuontrackingforcosmics1Leg (i.e. there is not the assumed muon id) and it is never used anywhere